core/src/main/java/org/mskcc/oncotree/model/MainType.java

```diff
@@ -23,7 +23,12 @@
   public MainType(){}
 
   public MainType(String name) {
-      this.name = name;
+    this.name = name;
+  }
+
+  public MainType(MainType otherMainType) {
```

Comment:
I guess this refactor is not yet in yet .. so this modification is still needed for now